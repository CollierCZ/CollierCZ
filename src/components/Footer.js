import styled from 'styled-components'
import React from "react"
import { Text, TextLink } from '@kiwicom/orbit-components'

const FooterContainer = styled.footer`
  margin: 1rem 0 0;
`

const Footer = ({author}) => {
  return (
    <FooterContainer><Text type="secondary">
        &copy; 2018&ndash;{new Date().getFullYear()}{" "}{author}. Powered by <TextLink
  href="https://www.gatsbyjs.org/"
  type="secondary"
>Gatsby</TextLink> and <TextLink
  href="https://kontent.ai"
  type="secondary"
>Kentico Kontent</TextLink>.</Text>
    </FooterContainer>
  );
}

export default Footer;
