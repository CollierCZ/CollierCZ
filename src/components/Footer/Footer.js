import React, { Component } from "react";
import "./Footer.css";

class Footer extends Component {
  render() {
    const { label } = this.props.copyrightLabel;
    const { year } = this.props.copyrightYear;
    return (
      <footer className="site-footer clearfix">
        <section className="copyright">
          &copy;{" "}{year || new Date().getFullYear()}{` ${label}`}
        </section>
      </footer>
    );
  }
}

export default Footer;
