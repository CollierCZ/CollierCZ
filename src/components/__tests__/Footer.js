import React from "react"
import { shallow } from "enzyme"
import Footer from "../Footer"

describe("Footer", () => {
  it("renders correctly", () => {
    const component = shallow (<Footer author="Aaron Collier" />)
    expect(component).toMatchSnapshot()
  })
})