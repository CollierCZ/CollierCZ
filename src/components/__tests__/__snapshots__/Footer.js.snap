// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="clearfix"
  css={
    Object {
      "map": undefined,
      "name": "m9pb97",
      "next": undefined,
      "styles": "
        position: relative;
        margin: 1rem 0 0;
        padding: 1rem 15px;
        font-family: \\"Open Sans\\", sans-serif;
        font-size: 1rem;
        line-height: 1.75em;
        color: #bbc7cc;
        @media only screen and (max-width: 500px) {
          margin-top: 0;
        }
        a {
          color: #bbc7cc;
          text-decoration: none;
          font-weight: bold;
        }
        a:hover {
          color: #bbc7cc;
          border-bottom: #bbc7cc 1px solid;
        }
      ",
      "toString": [Function],
    }
  }
>
  <section
    css={
      Object {
        "map": undefined,
        "name": "1odwksc",
        "next": undefined,
        "styles": "float:left;",
        "toString": [Function],
      }
    }
  >
    © 2018–
    2020
     
    Aaron Collier
    .
  </section>
  <section
    css={
      Object {
        "map": undefined,
        "name": "6n7j50",
        "next": undefined,
        "styles": "display:inline;",
        "toString": [Function],
      }
    }
  >
    Theme based on 
    <a
      href="https://github.com/TryGhost/Casper"
    >
      Casper
    </a>
    . Powered by 
    <a
      href="https://www.gatsbyjs.org/"
    >
      Gatsby
    </a>
     and 
    <a
      href="https://kontent.ai"
    >
      Kentico Kontent
    </a>
    .
  </section>
</footer>
`;
